modules/sdk-core/src/bitgo/utils/tss/baseTypes.ts

```diff
@@ -455,6 +457,8 @@ export interface MPCRecoveryOptions {
   walletPassphrase?: string;
   seed?: string;
   index?: number;
+  startingScanIndex?: number;
+  scan?: number;
```

Comment:
Please add jsdocs for these three fields. How is index different from scan?

Reply:
`index` and `startingScanIndex` both serve the same purpose, some WRW flows used `index` while others use `startingScanIndex`.